content/soarian/dstu2/infrastructure/documentReference.md

```diff
@@ -73,7 +68,7 @@ Notes:
 
 #### Request with attachment
 
-	GET https://fhir-myrecord.sandboxcerner.com/dstu2/123abc/DocumentReference?_format=json&attachment=docKey%3D101607B8CE55A4F20111E5B4F8D4AE526B7A48%26objNum%3D1%26created%3D20160301%26indexed%3D20160301&patientId=FaresRoni3425778"
+	GET https://fhir-myrecord.sandboxcerner.com/dstu2/123abc/DocumentReference?_format=json&attachment=docKey%3D101607B8CE55A4F20111E5B4F8D4AE526B7A48%26objNum%3D1%26created%3D20160301%26indexed%3D20160301&patientId=FCC941D7-60B9-491D-BEED-27629E47CD4E 
```

Comment:
I think I'm still not following/may be misunderstanding. What's the functional difference between this request and the request above?